assets/source/sass/layout/_header.scss

```diff
@@ -1,314 +1,256 @@
-/* === Header === */
```

Comment:
There's quite a bit of mixed indentation in this file!

(See `npm run sass-lint`.)

Reply:
Fixed in aa0af78